bumpversion/__init__.py

```diff
@@ -336,7 +342,59 @@ def __iter__(self):
         return iter(self._values)
 
     def __repr__(self):
-        return '<bumpversion.Version:{}>'.format(keyvaluestring(self._values))
+        by_part = ", ".join("{}={}".format(k, v) for k, v in self.items())
+        return '<bumpversion.Version:{}>'.format(by_part)
```

Comment:
Probably cleaner to use `self.__class__` to show the class name rather than hardcoding.

Reply:
Makes sense. Do notice that this is how it was done in the past, I just added the by_part line ;)